src/faction.h

```diff
@@ -136,7 +159,9 @@ class faction : public faction_template
         void serialize( JsonOut &json ) const;
         void faction_display( const catacurses::window &fac_w, int width ) const;
 
+
         std::string describe() const;
+        bool check_relations( std::vector<faction_power_spec> jo ) const;
```

Comment:
From the clang-tidy check
```suggestion
        bool check_relations( const std::vector<faction_power_spec> &faction_power_specs ) const;
```